java/lang/String/String.cpp

```diff
@@ -50,6 +50,11 @@ String::String(string target) {
 	this->size = length_pointer_char(target);
 }
 
+String::String(string original, int length) {

```

Comment:
Please add test case for String::String(string original, int length) in TEST (JavaLang, StringConstructor)